Runtime/Core/ActionResult.cs

```diff
@@ -61,12 +62,21 @@ public ActionResult(ActionResult.Status status, string notification)
             return res != null && res.status == Status.Success;
         }
 
+        /// <summary>
+        /// Convert the ActionResult to a boolean value, true if successful and false if not.
+        /// </summary>
+        /// <returns>True if action was successful, false otherwise.</returns>
         public bool ToBool()
         {
             return status == Status.Success;
         }
```

Comment:
Why does this function exist at all? There's an implicit cast right above this

Reply:
Yeah I guess this is an old remainder, not sure this is necessary anymore. However I guess changing this would be an API change?